src/components/app/Home.js

```diff
@@ -29,12 +38,8 @@ const InstallButton = ({
     <a
       href={xpiUrl}
       className={className}
-      onClick={e => {
-        if (window.InstallTrigger) {
-          window.InstallTrigger.install({ [name]: xpiUrl });
-        }
-        e.preventDefault();
-      }}
+      data-name={name}
```

Comment:
I feel like this would be clearer as a class and a bound method rather than passing data through `data-name`.

Also, see this "hot take" from Twitter that makes sense to me now: https://twitter.com/stevekinney/status/930841664452562944

Reply:
You're totally right, I changed this to a pure component. I'm not sure I agree to always use PureComponents but in this case it makes totally sense.